app/plugins/Wallet/js/buttons.js

```diff
@@ -106,7 +106,7 @@ addResultListener('coin-sent', function(result) {
 // Lock or unlock the wallet
 eID('lock-pod').onclick = function() {
 	var state = eID('lock-status').innerHTML;
-	if (!wallet.unlocked && state === 'Unencrypted') {
+	if (!wallet.encrypted && state === 'Create Wallet') {
 		encrypt();
```

Comment:
It more obvious now that it's just a single element and it has a big ol' `+` next to it